interpreter/cling/tools/packaging/cpt.py

```diff
@@ -276,10 +275,18 @@ def download_llvm_binary():
         raise Exception("Building clang using LLVM binary not possible. Please invoke cpt without --with-llvm-binary and --with-llvm-tar flags")
     if tar_required:
         llvm_flags = llvm_flag_setter(llvm_dir, llvm_config_path)
-        if DIST == "Ubuntu" and REV == '16.04' and is_os_64bit():
-            download_link = 'http://releases.llvm.org/5.0.2/clang+llvm-5.0.2-x86_64-linux-gnu-ubuntu-16.04.tar.xz'
+        if DIST == "Ubuntu" and REV == '20.04' and is_os_64bit():
+            download_link = 'https://github.com/llvm/llvm-project/releases/download/llvmorg-13.0.0/clang+llvm-13.0.0-x86_64-linux-gnu-ubuntu-20.04.tar.xz'
+            exec_subprocess_call('wget %s' % download_link, workdir)
```

Comment:
Didn't we replace the subprocess calls to the proper `wget` analog in python? Can that be moved to it as well?

Reply:
@vgvassilev Yeah just fixed it thank you